serpens/sqs.py

```diff
@@ -14,6 +14,47 @@
 
 logger = logging.getLogger(__name__)
 
+SQS_MESSAGE_BATCH_SIZE = 10
+
+
+def send_batch_message(queue_url, entries):
```

Comment:
Eu entendo que esse método deveria ser privado, ou seja, renomearia ele para _send_batch_message.

Reply:
Concordo: Delivers up to ten messages to the specified queue